src/avif.c

```diff
@@ -409,6 +409,66 @@ avifBool avifImageUsesU16(const avifImage * image)
     return (image->depth > 8);
 }
 
+uint8_t * avifImagePlane(const avifImage * image, int channel)
+{
+    if ((channel == AVIF_CHAN_Y) || (channel == AVIF_CHAN_U) || (channel == AVIF_CHAN_V)) {
+        return image->yuvPlanes[channel];
+    }
+    if (channel == AVIF_CHAN_A) {
+        return image->alphaPlane;
+    }
+    return NULL;
+}
+
+uint32_t avifImagePlaneRowBytes(const avifImage * image, int channel)
+{
+    if ((channel == AVIF_CHAN_Y) || (channel == AVIF_CHAN_U) || (channel == AVIF_CHAN_V)) {
+        return image->yuvRowBytes[channel];
+    }
+    if ((channel == AVIF_CHAN_A) && (image->alphaPlane)) {
```

Comment:
Nit: omit `&& (image->alphaPlane)`.

Note that we don't check `image->yuvPlanes[channel]`, which can also be null (for monochrome images).

Also, in the `avifImage` structs that we construct, I believe we maintain the invariant that if `image->alphaPlane` is null, then `image->alphaRowBytes` is 0. We also maintain a similar invariant for `image->yuvPlanes[channel]` and `image->yuvRowBytes[channel]`.